charts/metal3-deploy/0.1.0/charts/ironic/charts/mariadb/templates/deployment.yaml

```diff
@@ -25,6 +25,7 @@ spec:
       serviceAccountName: {{ include "mariadb.serviceAccountName" . }}
       securityContext:
         {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      hostNetwork: {{ .Values.useHostNetwork }}
```

Comment:
mariadb should not need to run on the hostNetwork ever AFAICS (since the image changes related to https://github.com/suse-edge/charts/pull/27 landed) - can we just access it via the service DNS name instead of doing this?

Reply:
You are correct, this seems to be a leftover from when the chart the mariadb deployment had the `hostNetwork` property hardcoded. As this is not needed it has been removed.